### ISSUE TITLE
Keep getting error while trying to install upgrade assistant

### ISSUE THREAD

**Issue by jladerma** [dotnet/modernize-dotnet#1284]
Problem encountered on https://dotnet.microsoft.com/en-us/platform/upgrade-assistant/tutorial/install-upgrade-assistant
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

I keep getting this error while trying to install the upgrade-assistant:
dotnet tool install -g --add-source 'https://api.nuget.org/v3/index.json' --ignore-failed-sources upgrade-assistant
C:\Users\XXXXX\AppData\Local\Temp\l0comugv.dnx\restore.csproj(5,31): error MSB4236: The SDK 'Microsoft.NET.Sdk' specified could not be found.
The tool package could not be restored.
Tool 'upgrade-assistant' failed to install. This failure may have been caused by:

* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
* You mistyped the name of the tool.

For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool

I've installed the latest SDK, rebooted, made sure my VS 2022 is up to date (17.3.3). NuGet works fine while installing items to solutions.

**Comment by brandonh-msft** [comment]
Did you follow the step immediately preceding? https://dotnet.microsoft.com/en-us/platform/upgrade-assistant/tutorial/install-sdk

**Comment by jladerma** [comment]
> Did you follow the step immediately preceding? https://dotnet.microsoft.com/en-us/platform/upgrade-assistant/tutorial/install-sdk

Yes, I did. I followed the steps in order and didn't have any failures.

**Comment by brandonh-msft** [comment]
The failure seems to point to not having the .NET 6 SDK on your system, can you paste the output of `dotnet --list-sdks` please? Thanks!

**Comment by jladerma** [comment]
> The failure seems to point to not having the .NET 6 SDK on your system, can you paste the output of `dotnet --list-sdks` please? Thanks!

![image](https://user-images.githubusercontent.com/102318107/188686668-60d3873e-e4d5-44ab-825b-2dc6d4304c9c.png)

**Comment by GeoDirk** [comment]
@jladerma  Had the same problem till I upgraded everything today and did a reboot:

Fully updated windows:  2022-09 Cumulative Update for Windows 11 for x64-based Systems (KB5017383)

Fully updated Visual Studio: version 17.3.4

Latest .NET SDK: version 6.0.401

It finally installed!

**Comment by jladerma** [comment]
@brandonh-msft I'm not sure why this was closed. It's still unresolved for me.

**Comment by brandonh-msft** [comment]
> @brandonh-msft Brandon Hurlburt FTE I'm not sure why this was closed. It's still unresolved for me.

good to know - have you fully updated as indicated by the other user who had the same problem?

**Comment by jladerma** [comment]
I am not able to change my version of windows as I'm on a corporate machine.  I last ran, the .NET SDK version was 6.0.400 and VS was 17.3.3, which were the latest at the time. I'm on windows 10 Enterprise 10.0.19044 Build 19044. If you would expect .NET SDK and VS versions to make a difference, I can upgrade and try again, but I cannot update the version of Windows

**Comment by brandonh-msft** [comment]
> I am not able to change my version of windows as I'm on a corporate machine. I last ran, the .NET SDK version was 6.0.400 and VS was 17.3.3, which were the latest at the time. I'm on windows 10 Enterprise 10.0.19044 Build 19044. If you would expect .NET SDK and VS versions to make a difference, I can upgrade and try again, but I cannot update the version of Windows

yeah let's get VS & .NET SDKs up-to-date just to remove it as a possible cause

**Comment by jladerma** [comment]
No, still get this:
C:\Users\xxxxxx\AppData\Local\Temp\kpn0ahlg.ghv\restore.csproj(5,31): error MSB4236: The SDK 'Microsoft.NET.Sdk' specified could not be found.
The tool package could not be restored.
Tool 'upgrade-assistant' failed to install. This failure may have been caused by:

* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
* You mistyped the name of the tool.

For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool

Which is odd since it also says:
Welcome to .NET 6.0!
---------------------
SDK Version: 6.0.401
